client/src/partials/finance/reports/cash_payment/cash_payment.js

```diff
@@ -64,8 +66,13 @@ function CashPaymentRegistryController(Cash, Notify, Session, Modal, uiGridConst
     { field : 'action', displayName : '...',
       cellTemplate: 'partials/finance/reports/cash_payment/templates/action.grid.html',
       enableFiltering: false
-    }
+    },
+    { field : 'action', displayName : '',
+      cellTemplate: 'partials/finance/reports/cash_payment/templates/cancelCash.action.tmpl.html',
+      enableFiltering: false
+    }  
   ];
+  vm.gridOptions.rowTemplate = '/partials/finance/reports/cash_payment/templates/grid.canceled.tmpl.html';
```

Comment:
:+1:  At some point, we should centralize these templates, so we can share them between billing services, Patient Invoices and Cash Payments.  It would be more maintainable.